obal/data/modules/srpm.py

```diff
@@ -54,6 +61,32 @@ def copy_sources(spec_file, package_dir, sources_dir):
             run_command(["git-annex", "lock", "--force"])
 
 
+def fetch_remote_sources(source_location, source_system, sources_dir):
+    """
+    Copy RPM sources from a remote source like Jenkins to rpmbuild environment
+    """
+    source_system_urls = {
```

Comment:
I thought more of a global, static var (`SOURCE_SYSTEM_URLS`), but wouldn't mind either way here.

Reply:
I was worried about scope creep, and I figured if I went this route and there was a reason to go with a global static variable you'd point it out to me :)